Configurations/README.md

```diff
@@ -480,12 +480,8 @@ For any name given by `build_file`, the "unified" system expects a
 template file in `Configurations/` named like the build file, with
 `.tmpl` appended, or in case of possible ambiguity, a combination of
 the second `build_scheme` list item and the `build_file` name.  For
-example, if `build_file` is set to `Makefile`, the template could be
-[`Configurations/Makefile.tmpl`](Makefile.tmpl) or
+example, if `build_file` is set to `Makefile`, the template would be
 [`Configurations/unix-Makefile.tmpl`](unix-Makefile.tmpl).
-In case both [`Configurations/unix-Makefile.tmpl`](Makefile.tmpl) and
-[`Configurations/Makefile.tmpl`](Makefile.tmpl) are present, the former takes
-precedence.
```

Comment:
Actually, having links here is pretty damn dumb.  The text should read like this:

```
the second `build_scheme` list item and the `build_file` name.  For
example, if `build_file` is set to `Makefile`, the template could be
`Configurations/Makefile.tmpl` or `Configurations/unix-Makefile.tmpl`.
In case both `Configurations/unix-Makefile.tmpl` and
`Configurations/Makefile.tmpl` are present, the former takes precedence.
```

Reply:
Someone was a bit over-zealous making links of everything...

Reply:
That makes sense, I'll update it. Thanks